lib/bitex_bot/settings.rb

```diff
@@ -1,37 +1,54 @@
 require 'hashie'
 
 module BitexBot
+  ##
+  # Documentation here!
+  #
   class FileSettings < ::Hashie::Clash
+    def respond_to_missing?(method_name, include_private)
+      super
+    end
+
     def method_missing(name, *args)
       return super unless args.size == 1 && args.none?
       self[name] = args.first
     end
   end
 
+  ##
+  # This class load settings file, else write a sample file.
+  #
   class SettingsClass < ::Hashie::Mash
     include ::Hashie::Extensions::Mash::SymbolizeKeys
 
     def load_default
-      path = ARGV[0] || 'bitex_bot_settings.rb'
-      unless FileTest.exists?(path)
-        sample_path = File.expand_path('../../../settings.rb.sample', __FILE__)
-        FileUtils.cp(sample_path, path)
-        puts "No settings found, I've created a new one with sample "\
-          "values at #{path}. Please go ahead and edit it before running this again."
-        exit 1
-      end
+      path = ARGV[0] || 'bitex_botss_settings.rb'
```

Comment:
There's a typo in `bitex_botss_settings.rb`. It should be `bitex_bot_settings.rb`.